src/utils/s3.js

```diff
@@ -21,7 +21,7 @@ export default class S3Upload {
       partEl.appendChild(etagEl);
       root.appendChild(partEl);
     });
-    return new window.XMLSerializer().serializeToString(root);
+    return root.outerHTML;
```

Comment:
This was changed because whatever test browser environment mocha is using does not support `window.XMLSerializer`.